cinema/views.py

```diff
@@ -0,0 +1,45 @@
+from django.http import HttpResponse, JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.parsers import JSONParser
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@csrf_exempt
+def movie_list(request: HttpResponse) -> JsonResponse:
+    if request.method == 'GET':
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return JsonResponse(serializer.data, safe=False)
+
+    elif request.method == 'POST':
+        data = JSONParser().parse(request)
+        serializer = MovieSerializer(data=data)
+        if serializer.is_valid():
+            serializer.save()
+            return JsonResponse(serializer.data, status=201)
+        return JsonResponse(serializer.errors, status=400)
```

Comment:
I'd advise using DRF's `Response` to generate responses here. That way you won't have to use `safe=False`.

Reply:
Thanks, good advice!